### PR TITLE
Fix: Adding callbacks to notification view model

### PR DESCRIPTION
**Pull request by yassine04e**
This PR focuses on adding callbacks to notification view model (only to the function send notification), so that we improve the coherency of the Toasts being displayed

### CODE REVIEW

File: app/src/main/java/com/github/se/travelpouch/model/notifications/NotificationViewModel.kt [hunk 1]
```diff
@@ -22,6 +21,7 @@ class NotificationViewModel
 @Inject
 constructor(private val notificationRepository: NotificationRepository) : ViewModel() {
 
+  // Preparing branch PR
```
Comment (RemIsMyWaifuu) [inline app/src/main/java/com/github/se/travelpouch/model/notifications/NotificationViewModel.kt:24]: ???

Comment (yassine04e) [reply app/src/main/java/com/github/se/travelpouch/model/notifications/NotificationViewModel.kt:24]: Yes I have to remove it.

**Comment by RemIsMyWaifuu** [comment]
You woke me up from my slumber, I hope it was worth it.

**Review by RemIsMyWaifuu — APPROVED** [review]
Works when tested on my device. Better Toasts now. Approved

**Pull request merged**